diff --git a/kahypar/application/CMakeLists.txt b/kahypar/application/CMakeLists.txt
index f62320c2..355cce63 100644
--- a/kahypar/application/CMakeLists.txt
+++ b/kahypar/application/CMakeLists.txt
@@ -1,6 +1,8 @@
 add_executable(KaHyPar kahypar.cc)
 target_link_libraries(KaHyPar ${Boost_LIBRARIES})
 
+install(TARGETS KaHyPar)
+
 set_property(TARGET KaHyPar PROPERTY CXX_STANDARD 17)
 set_property(TARGET KaHyPar PROPERTY CXX_STANDARD_REQUIRED ON)
 
